### DISCUSSION TITLE
Need help limiting the length of anchor on the left side

### DISCUSSION THREAD

**Discussion by tejab92** [Eliav2/react-xarrows#57] [Q&A]
I am using this library to build a feature where there is a tree structure and user can connect nodes. But the problem I'm facing is the arrows are getting rendered such that it is blocking other nodes. I would like to restrict the left anchor to a fixed length
![image](https://user-images.githubusercontent.com/82674602/115066362-44bd1680-9ea4-11eb-89be-2ea9c41f7908.png)

**Comment by tejab92** [comment]
@Eliav2

**Comment by Eliav2** [comment]
not sure what you mean. what exactly is the problem?

**Reply by tejab92** [reply]
Thanks for responding. Attached another screenshot of what I am expecting. I want the line to go along the path similar to the red line. 
![image](https://user-images.githubusercontent.com/82674602/115071461-23abf400-9eab-11eb-97b0-14a77dd7b2ab.png)

**Reply by tejab92** [reply]
@Eliav2 do you know if there is a solution to this? I can also poke around and see if the source code can be tweaked.

**Reply by Eliav2** [reply]
currently no.
will add a property to be able to choose "grid break" property.(currently the default is 50% always)

nice note friend. stay tuned for v1.7.0

**Reply by Eliav2** [reply]
thank you friend. 
see https://github.com/Eliav2/react-xarrows/blob/master/src/index.tsx#L505
note that there is curves for vertical to vertical and horizontal to horizontal. just update 0.5 to a new property with name as you please.
